app/src/main/res/values/strings.xml

```diff
@@ -330,4 +330,17 @@
     <string name="title_suggestions_holiday">Happy Holidays!</string>
     <string name="desc_suggestions_holiday">Try some of our holiday suggestions</string>
     <string name="lbl_latest_tv_shows">Latest TV Shows</string>
+    <string name="lbl_news">News</string>
+    <string name="lbl_new_only">New Only</string>
+    <string name="lbl_audio_delay">Audio Delay</string>
+    <string name="lbl_new_episodes">New Episodes</string>
+    <string name="lbl_premieres">Premieres</string>
+    <string name="lbl_repeat_episodes">Repeat Episodes</string>
+    <string name="lbl_live_broadcasts">Live Broadcasts</string>
+    <string name="lbl_hd_programs">HD Programs</string>
+    <string name="lbl_colored_backgrounds">Enable Color Coded Backgrounds</string>
+    <string name="lbl_start_favorites">Place favorites at beginning</string>
```

Comment:
I would use title casing for the favorite order string if it's a preference. Also, I would change the colored backgrounds preference to something like `Generate Background Color from Images` because the current string is pretty vague.

Reply:
I updated the casing, but the "color coded backgrounds" refers to the live tv guide feature where the background of the program differs depending on the genre or category. A very similar wording is used in web for this feature.

Reply:
Ah okay, I misunderstood the meaning. It still isn't very clear but I guess it's hard to explain what this option does. I have been removing the `Enable` prefix in front of boolean settings though since the checkbox already indicates the behavior.